modules/boot-api/pom.xml

```diff
@@ -20,38 +20,6 @@
     </properties>
 
     <dependencies>
-        <dependency>
-            <groupId>commons-cli</groupId>
-            <artifactId>commons-cli</artifactId>
-        </dependency>
-
-        <!-- Needed by the MuleLog4jConfigurer -->
-        <dependency>
-            <groupId>org.mule.runtime.boot</groupId>
-            <artifactId>mule-module-log4j-boot-configurator</artifactId>
-            <version>${project.version}</version>
-        </dependency>
-
-        <!-- Needed by the BootModuleLayerValidationBootstrapConfigurer and for creating the container ClassLoader -->
-        <dependency>
-            <groupId>org.mule.runtime.boot</groupId>
-            <artifactId>mule-module-jpms-utils</artifactId>
-            <version>${project.version}</version>
-        </dependency>
-
-        <!-- Needed by the SLF4JBridgeHandlerBootstrapConfigurer, but also to make the logging modules available from the boot layer -->
-        <dependency>
-            <groupId>org.mule.runtime.boot</groupId>
-            <artifactId>mule-module-logging</artifactId>
-            <version>${project.version}</version>
-        </dependency>
-
-        <dependency>
-            <groupId>org.mule.runtime</groupId>
-            <artifactId>api-annotations</artifactId>
-            <version>${muleAnnotationsApiVersion}</version>
-            <scope>provided</scope>
-        </dependency>
 
         <dependency>
             <groupId>org.mule.tests</groupId>
```

Comment:
A test `MuleContainerBootstrapUtilsTestCase` remains there since it's related to reboot, not boot. It doesn't compile once I remove this dependency so I removed this dependency and moved the test to boot-commons.